docs/cancelling.md

```diff
@@ -1,5 +1,22 @@
-## How to cancel a transfer?
+# Cancelling
+
+## How do I cancel a transfer?
+
+To cancel an Interac e-Transfer that you have sent, click the link in the notification email, and you will be taken to the cancellation page. From there you can review the transfer details, and use the cancel option provided.
 
 ## How long does it take?
 
-## Do you charge cancellation fees?
\ No newline at end of file
+Depending on which cryptocurrency you used for sending, 
+the transaction times vary and can take anywhere from 15 seconds to 1 hour. See [transaction time](/basics#transaction-time) for details.
```

Comment:
the transaction **time varies** and can take anywhere from 15 seconds to 1 hour.